src/nandini/ArrayOperations.java

```diff
@@ -0,0 +1,89 @@
+package nandini;
+
+public class ArrayOperations {
+	
+	  int getMaxNumber(int num[]) {
+		
+		int max = num[0];
+		for(int index=0; index < num.length; index++) {
+			
+			if(max < num[index])
+				max = num[index];
+		}
+		
+		return max;
+	 }
+	  
+	  int getMinNumber(int num1[]) {
+			
+			int min =num1[0];
+			for(int index=0; index < num1.length; index++) {
+				
+				if(min > num1[index])
+					min = num1[index];
+			
+			}
+			return min;
+		}
+	  
+	  double getAvgNumbers(int num2[]) {
+				
+		   double avgOfSum = 0;
+		   for(int index=0; index < num2.length; index++) {
+					
+			   avgOfSum = avgOfSum + num2[index]/5;
+				
+		   }
+				return avgOfSum;
+		}
+			
+	  void getLengthofArray(int num3[]) {
+				
+		   for(int index=0; index < num3.length; index++) {
+					
+				System.out.println(num3[index]);
+				
+		   }	
+		       
+		}
+			
+       void getNumbersInReverse(int num4[]) {
+				
+		   for(int index= num4.length-1; index >= 0; index--) {
+					
+					System.out.println(num4[index]);
+				}
+				     
+			}
+			
+        double getMinMaxAvg(int maxNum, int minNum) {
+            	
```

Comment:
I think we have requirement where you need to find max & min number then you need to find avg of both, so this expecting two variable in method looks wrong as per the implementation perspective, user will give us array, not min & max value. I hope you got my point.


You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2